ci/ci_setup.sh

```diff
@@ -14,3 +14,7 @@ rm -rf spec/reports # no stale spec reports from previous executions
 
 # Setup the environment
 rake bootstrap # Bootstrap your logstash instance
+
+# Set up some general options for the rspec runner
+echo "--order rand" > .rspec
+echo "--format CI::Reporter::RSpec" >> .rspec
```

Comment:
I would like to keep the rake test tasks options free as much as possible, specially for the formatters, etc... but just my way of thinking.